esp/esp/dbmail/cronmail.py

```diff
@@ -42,6 +42,10 @@
 
 from django.conf import settings
 
+
+_ONE_WEEK = timedelta(weeks=1)
+
+
 @transaction.autocommit
```

Comment:
@benjaminjkraft, can you confirm that it is correct to leave this here? In main, there is no decorator. But you didn't remove this during your backport.